src/bp/ui-shared/src/Contents/Components/Form/index.tsx

```diff
@@ -259,6 +293,7 @@ const Form: FC<FormProps> = ({ bp, overrideFields, contentType, formData, fields
       default:
         return (
           <FieldWrapper key={field.key} label={printLabel(field, data[field.key])}>
+            {field.moreInfo && printMoreInfo(field.moreInfo)}
```

Comment:
You have the same logic for each fields, what do you think of moving the null check in printMoreInfo instead? 
So you can simply use `{printMoreInfo(field.moreInfo)}`